setup.py

```diff
@@ -46,7 +46,8 @@ def is_rtd() -> bool:
 
 
 install_requires = [
-    "overrides",
+    # see https://github.com/mkorpela/overrides/issues/121
+    "overrides<7.6",
```

Comment:
I'm guessing this is sufficiently breaking that it'll be fixed in 7.6.1, so maybe we want:

```suggestion
    "overrides!=7.6.0",
```